deploy/scripts/aad-register.ps1

```diff
@@ -246,6 +247,7 @@ Function New-ADApplications() {
                     $user = (Get-AzureADUser -SearchString $creds.Account.Id)[0]
                 }
                 catch {
+                    Write-Host "Getting user principal for $($creds.Account.Id) failed when searching by the acount ID."
```

Comment:
Fix typo "acount" and avoid passive voice, e.g. "Failed getting user principal for $($creds.Account.Id) while searching by account ID"

Reply:
Thanks and great callout!  Will update.